server/config.ts

```diff
@@ -58,6 +58,22 @@ export default {
       systemClientId: get('SYSTEM_CLIENT_ID', 'clientid', requiredInProduction),
       systemClientSecret: get('SYSTEM_CLIENT_SECRET', 'clientsecret', requiredInProduction),
     },
+    prison: {
+      url: get('PRISON_API_URL', 'http://localhost:8080', requiredInProduction),
+      timeout: {
+        response: Number(get('PRISON_API_TIMEOUT_RESPONSE', 10000)),
```

Comment:
Do we need to worry about if an env is present but it is not a valid number?

Reply:
I think it would be unlikely, it'll have to get set in the helm files and reviewed.  It defaults to 1000 if one isn't set.